### PR TITLE
Improve the -j option; set default value to 1

### PR DESCRIPTION
**Pull request by erezsh**

### CODE REVIEW

File: data_diff/__main__.py [hunk 1]
```diff
@@ -44,7 +44,12 @@
 @click.option("-v", "--verbose", is_flag=True, help="Print extra info")
 @click.option("-i", "--interactive", is_flag=True, help="Confirm queries, implies --debug")
 @click.option(
-    "-j", "--threads", default=None, help="Number of threads to use. 1 means no threading. Auto if not specified."
+    "-j",
+    "--threads",
+    default="1",
+    help="Number of worker threads to use per database. Default=1. "
```
Comment (sirupsen) [inline data_diff/__main__.py:50]: Would it be worth adding some more information here about tuning of this?

```
"Number of worker threads to use per database to execute queries simultaneously. Default = 1"
"A higher number will increase performance, but take more capacity from your database."
"Tune this while monitoring your database."
```

File: data_diff/__main__.py [hunk 2]
```diff
@@ -44,7 +44,12 @@
...
+    "'auto' will use the maximum amount recommended by Python. "
```
Comment (sirupsen) [inline data_diff/__main__.py:51]: What does "recommended by Python" mean? 👂🏻

**Review by sirupsen — COMMENTED** [review]
Just some questions and suggestions, but if this block you tomorrow, you can merge and address in follow-ups

File: data_diff/__main__.py (hunk 2 above)
Comment (erezsh) [reply data_diff/__main__.py:51]: It usually means equal to the number of cores on your local computer.

Comment (erezsh) [reply data_diff/__main__.py:51]: (I know it's not necessarily the best option for database workers, but at least it keeps it within a reasonable range)

File: data_diff/__main__.py (hunk 1 above)
Comment (erezsh) [reply data_diff/__main__.py:50]: It's definitely good information to provide. The question is should this be part of the help text, or somewhere in the docs?

Comment (sirupsen) [reply data_diff/__main__.py:50]: I think at least the first line in the help text, if you think it's all too long.

File: data_diff/__main__.py (hunk 2 above)
Comment (sirupsen) [reply data_diff/__main__.py:51]: Yeah it's a bit weird... I think we should just remove `auto` then 🤷🏻 I think it's more confusing than valuable.

Comment (erezsh) [reply data_diff/__main__.py:51]: Fair enough

**Review by sirupsen — APPROVED** [review]

**Pull request merged**